sdk/src/permission.ts

```diff
@@ -0,0 +1,54 @@
+import {id} from '@ethersproject/hash';
+
+export enum Operation {
```

Comment:
please add the value, TS does automatically but it depends on the order of the enum and is not totally clear

Reply:
Good! Applied in d0fd87f

Here is the source of truth for double-checking:
https://github.com/aragon/osx/blob/1dd896385225e55d05ad400a4761840b9d6d3254/packages/contracts/src/core/permission/PermissionLib.sol#L17-L21